modules/calc/src/main/java/com/opengamma/strata/calc/runner/CalculationResult.java

```diff
@@ -31,8 +23,8 @@
  * This stores the calculated result for a single cell in the output grid.
  * A set of related results for a single target can be stored in a {@link CalculationResults} instance.
  */
-@BeanDefinition(builderScope = "private")
-public final class CalculationResult implements ImmutableBean, Serializable {
```

Comment:
Why remove `Serializable`? It seems reasonable for result types to be serializable.